setup.py

```diff
@@ -38,12 +38,35 @@ def read_version():
     "numpy>=1.9.0",
     "protobuf>=3.1",
     "scipy>=0.19.0",
-    "urllib3>=1.21, <1.25",
     "protobuf3-to-dict>=0.1.5",
     "requests>=2.20.0, <2.21",
```

Comment:
I realize this isn't strictly in scope, but the request bounds were somewhat defined by docker-compose - we could probably change it to `requests>=2.20.0`?
* https://github.com/aws/sagemaker-python-sdk/pull/533
* https://github.com/aws/sagemaker-python-sdk/pull/538

Reply:
Maybe, don't have bandwidth to do a proper check for it as part of this PR knowing how many problems it caused before.

Reply:
fair enough 😂